python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -6652,3 +6652,78 @@ def test_get_pool_limits(
         assert limit.name == "bar"
         assert limit.limit == 1
         assert limit.from_default
+
+    def test_fetch_failed_materializations(self, test_run_id, storage: EventLogStorage):
```

Comment:
this test is going to fail until you either skip it in certain storages or implement  fetch_failed_materializations in every storage right?

Reply:
yeah i'm going back and forth on whether i want to do this or gate calling this in the GQL resolver on `instance._event_storage.asset_records_have_last_planned_and_failed_materializations` 
and add a some pyright ignores

Reply:
actually i think it should be fine because i implemented it in `SqlEventLogStorage` which is the parent class for all of the OSS implementations